javascript/ql/lib/semmle/javascript/internal/ConceptsShared.qll

```diff
@@ -0,0 +1,13 @@
+/**
+ * Provides Concepts which are shared across languages.
+ *
+ * Each language will have a language specific `Concepts.qll` file that can import the
```

Comment:
```suggestion
 * Each language has a language specific `Concepts.qll` file that can import the
```
I think this is clearer to the reader, who might wonder if it describes the current state or some future plan that hasn't yet been realised.

Reply:
Done in [414764c](https://github.com/github/codeql/pull/8476/commits/414764cceeb77b7072da392b2e5474077b953061)

Reply:
(also, this direct style matches some rule in our doc style guide)